torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2685,6 +2685,90 @@ def generator():
     return list(generator())
 
 
+def sample_inputs_nn_pad(op_info, device, dtype, requires_grad, mode, **kwargs):
+    assert mode in ('constant', 'reflect', 'replicate', 'circular')
+    if mode in ['reflect', 'replicate']:
+        cases: tuple = (  # ignore
+            ((1, 3), (1, 2)),
+            ((1, 3), (0, 1)),
+            ((0, 3, 3), (1, 2)),
+            ((0, 3, 3), (0, 1)),
+            ((1, 3, 3), (1, 2)),
+            ((1, 3, 3), (0, 1)),
+            ((1, 3, 3), (0, 2, 0, 1)),
+            ((0, 3, 3, 3), (0, 2, 0, 1)),
+            ((3, 3, 5, 5), (0, 2, 0, 1)),
+            ((3, 3, 5, 5), (1, 1, 1, 1, 1, 1)),
+            ((1, 3, 3, 3, 3), (1, 1, 1, 1, 1, 1)),
```

Comment:
Looks like the docs on reflect padding are wrong...
![image](https://user-images.githubusercontent.com/5652049/128932728-1c79e583-9b41-4e22-94b2-81e513a778ba.png)

Reply:
This has been fixed on master.
![image](https://user-images.githubusercontent.com/19503980/128953857-45333ada-762a-474e-b9bc-c3e42af13632.png)